01week/rockPaperScissors.js

```diff
@@ -1,51 +1,52 @@
-'use strict';
+"use strict";
 
-const assert = require('assert');
-const readline = require('readline');
+const assert = require("assert");
+const readline = require("readline");
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  if (hand1 === hand2) return "it's a tie";
+  else if (hand1 === "rock" && hand2 === "scissors") return "Hand one wins!";
```

Comment:
Try writing this if statement like this:
if ("condition is met"){
return "this thing" 
} else if ("other condition"){
return "other thing"
}